KsApi/PerimeterXClientType.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+public protocol PerimeterXClientType: PerimeterXErrorHandler {
+  /**
+   Returns a dictionary of `[String: String]`, representing httpHeaders from Perimeter X.
+   */
+  func headers() -> [String: String]
+}
+
+public protocol PerimeterXErrorHandler {
```

Comment:
What do you think about creating another file for this and removing the Perimeter X prefix? I don't think this protocol is specific to the perimeter x, but open to go either way

Reply:
Makes sense. Moved this into a separate file and removed the prefix in the latest commit.